1-exercises/F-array-forEach/exercise.js

```diff
@@ -8,7 +8,17 @@
 */
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+arr.forEach(function(number){
+  let result = "";
 
+  if(number % 3 === 0){ result += `'Fizz'`}
+  if(number % 5 === 0){ result += `'Buzz'`}
+
+
+  if(result === "") {result = number;}
+
+  console.log(result);
+})
```

Comment:
same comment as above; this is fine but declare your function outside the scope of the `.forEach` and reference it, or use a arrow function.